views/viewing_rooms.erb

```diff
@@ -1,5 +1,8 @@
-Welcome to Makers BnB!<br/>
```

Comment:
Could we move the the rooms views to their own folders and name them like the routes? I think this would it make it easier to find them.

Reply:
done